index.js

```diff
@@ -330,20 +347,29 @@ function beforeQuit (e) {
   }, 300)
 
   // 'close' event will gracefully close databases and wait for pending sync
-  // TODO: Show the user that a sync is pending finishing
-  logger.debug('ipc.send close')
   ipc.send('close', null, () => {
     logger.info('IPC closed')
     clearTimeout(closingTimeoutId)
-    if (serverProcess) serverProcess.kill()
     try { closingWin.close() } catch (e) {}
     closingWin = null
-    serverProcess = null
+    killZombies()
     app.exit()
   })
 }
 
+function killZombies () {
```

Comment:
I know it's technically an orphan but I feel bad about killing them

Reply:
Computer scientist terminology, huh? Maybe we `resolve` the orphan? :P